sut/drush/Commands/SimpleSutCommands.php

```diff
@@ -42,7 +42,7 @@ public function example()
      *
      * @hook init sut:simple
      */
-    public function customLogger(DrushArgvInput $argv, AnnotationData $annotationData): void
+    public function customLogger(ArgvInput $argv, AnnotationData $annotationData): void
```

Comment:
Looks like we have a breaking change here (which hooks are affected?). Just noting it for release notes.

Reply:
I don't think this is a breaking change. If it is, we can put back DrushArgvInput and make it an empty extension of ArgvInput.

Reply:
I guess at a minimum this is a breaking change for anyone who copied `DrushArgvInput` from this code into their hook. I think that technically, though, using `DrushArgvInput` or even `ArgvInput` as I have here is a bug. It should be `InputInterface`, so that there isn't a type error if a non-ArgvInput $input is injected. I don't think it's possible for $input to be incompatible with ArgvInput, but it will be incompatible with DrushArgvInput if `--strict=0` is used.

Maybe worth putting into the release notes, but I don't think it's absolutely necessary to maintain b/c in Drush 11. If you want to, though, we can do what I said above and put an empty DrushArgvInput class for Drush 11, and remove in 12. That's pretty low effort / low impact, and the most conservative thing to do.